### PR TITLE
Fixed quantizehigh error; adapted test

### PR DESCRIPTION
**Pull request by ernstdevreede**
Fixes an error in quantizehigh for times, where a time that would round up to the next hour would be incorrectly calculated. Fixes NOAA GOES-16 images missing on every HH:00 except 00:00

### CODE REVIEW

File: tests/AdagucTests/TestWMS.py [hunk 1]
```diff
@@ -1569,7 +1569,7 @@ def test_WMSGetMapQuantizeHigh(self):
 
         filename = "testWMSGetCapabilities_quantizehigh.xml"
         status, data, headers = AdagucTestTools().runADAGUCServer(
-            "DATASET=adaguc.tests.quantizelow&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities",
+            "DATASET=adaguc.tests.quantizehigh&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities",
```
Comment (maartenplieger) [inline tests/AdagucTests/TestWMS.py:1572]: Where is the quantizelow test?

File: CCDFDataModel/CTime.cpp [hunk 2]
```diff
@@ -925,9 +926,12 @@ double CTime::quantizeTimeToISO8601(double offsetOrig, CT::string period, CT::st
       date.minute = origM - restM;
       offsetLow = thisTime->dateToOffset(date);
       if (restM > 0) {
-        date.minute = date.minute + M;
+        // date.minute = date.minute + M;
```
Comment (maartenplieger) [inline CCDFDataModel/CTime.cpp]: Can this be removed?

File: CCDFDataModel/CTime.cpp [hunk 3]
```diff
@@ -925,9 +926,12 @@ double CTime::quantizeTimeToISO8601(double offsetOrig, CT::string period, CT::st
...
+        offsetHigh = offsetLow + 60 * M;
+      } else {
+        offsetHigh = offsetLow;
       }
-      offsetHigh = thisTime->dateToOffset(date);
+      // offsetHigh = thisTime->dateToOffset(date);
```
Comment (maartenplieger) [inline CCDFDataModel/CTime.cpp]: Also remove?

File: tests/AdagucTests/TestWMS.py (hunk 1 above)
Comment (ernstdevreede) [reply tests/AdagucTests/TestWMS.py:1572]: That is in test_WMSGetMapQuantizeLow.
The test_WMSGetMapQuantizeHigh test erroneously used quantizelow configuration

File: CCDFDataModel/CTime.cpp (hunk 3 above)
Comment (ernstdevreede) [reply CCDFDataModel/CTime.cpp]: Removed

File: CCDFDataModel/CTime.cpp (hunk 2 above)
Comment (ernstdevreede) [reply CCDFDataModel/CTime.cpp]: Removed

**Review by maartenplieger — APPROVED** [review]

**Pull request merged**